### ISSUE TITLE
Consensus always 0%

### ISSUE THREAD

**Issue by simonmorgenthaler** [LiskHQ/lisk#1832]
### Expected behavior
`consensus` in the log and on the /api/node/status endpoint should return the actual value

### Actual behavior
it's always 0 in my log and also in my /api/node/status endpoint. Others reported that there is a real value returned, but it never changes.

### Steps to reproduce
Call /api/node/status on a 1.0.0-beta.2 node with an open api or check the logs

### Which version(s) does this affect? (Environment, OS, etc...)
lisk-1.0.0-beta.2-Linux-x86_64

**Comment by simonmorgenthaler** [comment]
consensus 0% in the log

**Comment by SargeKhan** [comment]
When node is node forging and receiving new blocks at the tip of the chain, it doesn't recalculate its own broadhash consensus. Due to which 0% consensus is returned. Which is not actually correct. Thanks for the creating the issue. :)

**Comment by simonmorgenthaler** [comment]
Please open again, with beta6 the consensus is still always the same, 0% right now

**Comment by SargeKhan** [comment]
@simonmorgenthaler @punkr0ck, are you running your instances with a publicly accessible IP? Because if not, then peers can't send you any information at all (which means that they are never able to notify you about their node status, or send new blocks or unconfirmed transactions), which triggers sync function repeatedly, and during sync operation, since peers list has outdated information (no peer was able to notify your peer about its node status), you get the log that broadhash consensus is 0%.

**Comment by simonmorgenthaler** [comment]
yes, ports are open. I now missed two blocks because of low consensus. A problem we know now since the very beginning of Lisk.
```
-A INPUT -p tcp -m tcp --dport 5000 -j ACCEPT
-A INPUT -p tcp -m tcp --dport 5001 -j ACCEPT
```

**Comment by SargeKhan** [comment]
Thanks @simonmorgenthaler, I am reopening this issue and will further investigate this problem.

**Comment by simonmorgenthaler** [comment]
still does not work properly in beta7. Please open again.

- On a non-forging node, consensus still is 0%. It also needs to be accurate on non-forging nodes.
- it updates more often than before, but not in a regularly manner. And much less often than every 5 sec.
